src/fhirpath/fhirspec/spec.py

```diff
@@ -23,6 +23,18 @@
 skip_because_unsupported = [r"SimpleQuantity"]
 HTTP_URL = re.compile(r"^https?://", re.IGNORECASE)
 
+search_param_prefixes: Dict[str, List[str]] = {
+    "eq": ["number", "date", "quantity"],
+    "ne": ["number", "date", "quantity"],
+    "gt": ["number", "date", "quantity"],
+    "lt": ["number", "date", "quantity"],
+    "ge": ["number", "date", "quantity"],
+    "le": ["number", "date", "quantity"],
+    "sa": ["number", "date", "quantity"],
+    "eb": ["number", "date", "quantity"],
+    "ap": ["number", "date", "quantity"],
```

Comment:
Aren't all the prefix applied to these types? If so, why do we need them here?

Reply:
I thought that the use of prefix may vary depending on search param types but indeed it seems to be always the same, fixing this.

Reply:
done